types/moment-timezone/index.d.ts

```diff
@@ -55,7 +54,15 @@ declare module "moment" {
         names(): string[];
         guess(ignoreCache?: boolean): string;
 
-        setDefault(timezone?: string): MomentTimezone;
+        setDefault(timezone: string): MomentTimezone;
+
+        pack(data: object): string;
```

Comment:
Documentation states the parameter is exactly like the Zone object. You might want to change the signature to use it instead.